src/media/css/constants/colors.styl

```diff
@@ -1,20 +1,56 @@
+/* 
+ * colors.styl -- contains variables for each color used in CSS.
+ *
+ * Any time you would reference a color in CSS, you should instead be using a
+ * variable defined in either this file or a project-specific colors file. This
+ * allows for consistency and predictability in color definition and use.
+ * 
+ * Colors are grouped into palettes related by intention or purpose. It is
+ * encouraged for palettes to reference other palettes' colors. For example,
+ * if the active navigation item's underline uses the primary palette's orange,
+ * you might define it this way:
+ * 
+ * $color--navigation--active-underline = $color--primary--orange;
+ *
+ * This provides flexibiltiy in changes to our use of colors; if we want to
+ * globally redefine the orange color in the primary palette,
+ * $color--primary--orange can be redefined. If we only want to change the
+ * active navigation underline color, $color--navigation--active-underline can
+ * be changed.
+ *
+ * Variable names should follow the form $color--pallete-name--color-name
+ */
+
 // Primary palette
-$primary-blue = #3A5990;
-$primary-teal = #00C9A8;
-$primary-green = #8BC873;
-$primary-yellow = #FAD73B;
-$primary-orange = #ED8040;
-$primary-red = #D46952;
-$primary-pink = #E77286;
-$primary-purple = #8D4B87;
+$color--primary--blue = #3A5990;
+$color--primary--teal = #00C9A8;
+$color--primary--green = #8BC873;
+$color--primary--yellow = #FAD73B;
+$color--primary--orange = #ED8040;
+$color--primary--red = #D46952;
+$color--primary--pink = #E77286;
+$color--primary--purple = #8D4B87;
 
 // Greyscale palette
-$greyscale-black = #1A1A1A;
-$greyscale-darkest = #535353;
-$greyscale-darker = #797979;
-$greyscale-dark = #B5B5B5;
-$greyscale-medium = #CBCBCB;
-$greyscale-light =  #D8D8D8;
-$greyscale-lighter = #F1F1F1;
-$greyscale-lightest = #F5F5F5;
-$greyscale-white = #FFF;
+$color--greyscale--black = #1A1A1A;
```

Comment:
Rather than a darkest to lighest, the greyscales should come in threes as described on http://marketplaceux.github.io/marketplace-style-guides/palette.html (except for white). A grey, with a dark version and a light version.